### PR TITLE
Bugfix, when receiving concatenated messages from a stream.

### PR DESCRIPTION
**Pull request by xzores**
Bugfix, when receiving concatenated messages from a stream.

### CODE REVIEW

**Comment by hootnot** [comment]
Hi @xzores , 
I'll have a look and come back to it.

It looks like you make some first steps coding Python since your using ';' to end statements. Use *pycodestyle / flake8* to style your code (in general)

**Comment by xzores** [comment]
Yeah, I come from C++

File: saxo_openapi/contrib/ws/stream.py [hunk 1]
```diff
@@ -20,25 +20,41 @@ def decode_ws_msg(raw):
...
-    (Srefid,), raw = struct.unpack_from("B", raw[:1]), raw[1:]
-    FS = "{}s".format(Srefid)
-    (refid,), raw = struct.unpack_from(FS, raw[:Srefid]), raw[Srefid:]
-    (payloadFmt,), raw = struct.unpack_from("B", raw[:1]), raw[1:]
-    (payloadSize,), raw = struct.unpack_from("i", raw[:4]), raw[4:]
+    msgList = [];
+    done : bool = False;
+    offset : int = 0;
+
+    print(len(rawData))
```
Comment (RusEu) [inline saxo_openapi/contrib/ws/stream.py:28]: :eyes:

**Comment by hootnot** [comment]
@xzores @RusEu 
I've fixed the issue and it now processes all messages instead of only the first one (test extended).

I choose to make it a generator instead of returning a list (as in this PR).

I will close this issue/pr now.

Thanks

**Pull request closed without merging**